src/spox/_schemas.py

```diff
@@ -14,20 +14,21 @@
 
 from onnx.defs import OpSchema, get_all_schemas_with_history
 
-
-class _Comparable(Protocol):
-    def __lt__(self, other): ...
-
-    def __gt__(self, other): ...
-
-
 S = TypeVar("S")
 K = TypeVar("K")
 V = TypeVar("V")
-T = TypeVar("T", bound=_Comparable)
+T = TypeVar("T", bound="_Comparable", contravariant=True)
+
+
+class _Comparable(Protocol[T]):
+    def __lt__(self: T, other: T) -> bool: ...
+
+    def __gt__(self: T, other: T) -> bool: ...
```

Comment:
This looks a little odd. The bound on `"_Comparable"` is implicitly a `"_Comparable[Any]"` anyhow. Let's just leave this largely as it was; only annotating `other: Any` and the return as `bool` without generics.